lib/generators/glueby/contract/templates/utxo_table.rb.erb

```diff
@@ -0,0 +1,16 @@
+class CreateUtxo < ActiveRecord::Migration<%= migration_version %>
+  def change
+    create_table :utxos<%= table_options %> do |t|
+      t.string     :txid
+      t.integer    :index
+      t.bigint     :value
+      t.string     :script_pubkey
+      t.boolean    :spent
+      t.integer    :status
+      t.belongs_to :key, null: true
+      t.timestamps
+    end
+
+    add_index :utxos, [:txid, :index]
```

Comment:
Isn't it better to have a unique constraint on txid and index?

Reply:
Added at 8ca322ed89ab1bb551e953f8b6d52cfe68490e4f